draft-ietf-mls-protocol.md

```diff
@@ -260,48 +262,48 @@ This protocol is designed to execute in the context of a Messaging Service (MS)
   in the same order to all participants.  (See {{sequencing}} for further
   considerations.)
 
-* A directory to which participants can publish initialization keys, and from which
-  participant can download initialization keys for other participants.
+* A directory to which clients can publish initialization keys and download
+  initialization keys for other participants.
 
 
 # Protocol Overview
 
-The goal of this protocol is to allow a group of participants to exchange confidential and
-authenticated messages. It does so by deriving a sequence of secrets and keys known only to group members. Those
+The goal of this protocol is to allow a group of clients, members of a group, to exchange confidential and
+authenticated messages. It does so by deriving a sequence of secrets and keys known only to members. Those
 should be secret against an active network adversary and should have both forward and
 post-compromise secrecy with respect to compromise of a participant.
 
-We describe the information stored by each participant as a _state_, which includes both public and
-private data. An initial state, including an initial set of participants, is set up by a group
-creator using the _Init_ algorithm and based on information pre-published by the initial members. The creator
-sends the _GroupInit_ message to the participants, who can then set up their own group state and derive
-the same shared secret. Participants then exchange messages to produce new shared states which are
+We describe the information stored by each client as a _state_, which includes both public and
+private data. An initial state, including an initial set of clients, is set up by a group
+creator using the _Init_ algorithm and based on information pre-published by clients. The creator
+sends the _GroupInit_ message to the clients, who can then set up their own group state and derive
+the same shared secret. Clients then exchange messages to produce new shared states which are
 causally linked to their predecessors, forming a logical Directed Acyclic Graph (DAG) of states.
-Participants can send _Update_ messages for post-compromise secrecy and new participants can be
-added or existing participants removed from the group.
+Members can send _Update_ messages for post-compromise secrecy and new clients can be
+added or existing members removed from the group.
 
-The protocol algorithms we specify here follow. Each algorithm specifies both (i) how a participant
-performs the operation and (ii) how other participants update their state based on it.
+The protocol algorithms we specify here follow. Each algorithm specifies both (i) how a client
+performs the operation and (ii) how other clients update their state based on it.
 
 There are four major operations in the lifecycle of a group:
 
 * Adding a member, initiated by a current member;
-* Adding a member, initiated by the new member;
+* Adding a member, initiated by the future member;
```

Comment:
I'll remove that line in a separate commit, since we currently don't have UserAdd

Reply:
I did that as well.